api/api.yaml

```diff
@@ -1162,6 +1162,120 @@ paths:
               schema:
                 $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
       x-codegen-request-body-name: installationUpdate
+  /projects/{projectIdOrName}/integrations/{integrationId}/installations/{installationId}:pause:
+    post:
+      summary: Pause reads for an installation
+      description: Pauses all reads for an installation. If reads are already paused, this will be a no-op. Accepts specific objects to pause reads for in the request body.
+      operationId: pauseReads
+      tags: ["Installation"]
+      parameters:
+        - name: projectIdOrName
+          in: path
+          required: true
+          description: The Ampersand project ID or project name.
+          schema:
+            type: string
+          example: my-project
+        - name: integrationId
+          in: path
+          required: true
+          description: The integration ID.
+          schema:
+            type: string
+        - name: installationId
+          in: path
+          required: true
+          description: The Ampersand installation ID.
+          schema:
+            type: string
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                objects:
+                  type: array
+                  items:
+                    type: string
+                    description: The names of the objects to pause reads for.
+                    example: ["account", "contact"]
+                note:
```

Comment:
I like the idea, but can you first verify that the Temporal API allows us to retrieve this note later easily? Since if the builder is allowed to specify a note, then they would expect to be able to retrieve it later.

Reply:
Yes, describing a schedule gives us the note that was last set on the workflow